cypress/e2e/6SentimentAnalysis.cy.js

```diff
@@ -0,0 +1,47 @@
+describe("Sentiment Analysis Test", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("should enter positive text, click Sentiment, and check for a positive result", () => {
+    // Type text into the textarea
+    cy.get("#inputted-text").type("I am delighted to announce that this test was a success!");
+
+    // Click the Sentiment button
+    cy.get("button").contains("Sentiment").click();
+
+    // Wait for the API call and result to appear (you may need to adjust the wait time)
+    cy.wait(3000); // Adjust the wait time as needed
+
+    // Assert that the sentiment result is displayed
+    cy.get("#sentiment-result").should("contain", "Positive");
```

Comment:
Just wondering if this should include the percentages? not sure if this can be added to the cypress testing.

Reply:
Added percentage checking